ci/pipeline.yml

```diff
@@ -547,8 +547,6 @@ jobs:
       stemcells:
       - opensearch-stemcell-jammy/*.tgz
       ops_files:
-      - deploy-logs-opensearch-config/opsfiles/remove-curator.yml
-      - deploy-logs-opensearch-config/opsfiles/remove-smoke-tests.yml
```

Comment:
Can we just delete these two files? I don't imagine we'll want to disable either of these jobs going forward